pkgs/development/ocaml-modules/ppxlib/default.nix

```diff
@@ -66,7 +66,7 @@ buildDunePackage rec {
   pname = "ppxlib";
   inherit version;
 
-  duneVersion = if param.useDune2 or true then "2" else "1";
+  duneVersion = if param.useDune2 or true then "3" else "1";
```

Comment:
If I am reading this correctly this is always 3?

Reply:
I think (hope) this says "if useDune2 is set, use that value OR true"

Reply:
Ah ok. That is hard to read, but I guess only temporary. I think I asked this before, but isn't `useDune2 = true` -> 3 a bit strange?